kornia/geometry/quaternion.py

```diff
@@ -273,7 +272,7 @@ def from_axis_angle(cls, axis_angle: Tensor) -> 'Quaternion':
             Parameter containing:
             tensor([[0.8776, 0.4794, 0.0000, 0.0000]], requires_grad=True)
         """
-        return cls(angle_axis_to_quaternion(axis_angle, order=QuaternionCoeffOrder.WXYZ))
+        return cls(angle_axis_to_quaternion(axis_angle))
```

Comment:
unrelated to this PR but I think we should probably correct the naming of this to axis_angle(and from other occurrences)

Reply:
agree